lib/cmd/common/build.js

```diff
@@ -48,7 +48,9 @@ function build(argv, cb) {
       validateArgsNgui(argObj);
     }
     mutateGitRefArgs(argObj);
-    doBuild(argObj, cb);
+    addCredentialsParams(argObj, function(){
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png) Add a semicolon at the end of this statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3ASemicolon)